TOC.md

```diff
@@ -2,187 +2,204 @@
+    + TiSpark
+      - [Quick Start Guide](tispark/tispark-quick-start-guide.md)
+      - [User Guide](tispark/tispark-user-guide.md)
+  + Configuration
+    + tidb-server
+      - [MySQL Compatibility](sql/variable.md)
+      - [Time Zone](sql/time-zone.md)
+      - [Character Set](sql/character-set-support.md)
+      - [Extensions to MySQL](sql/tidb-specific.md)
+      - [Memory Control](sql/tidb-memory-control.md)
+      - [Garbage Collection (GC)](op-guide/gc.md)
+      - [Configuration Flags](op-guide/configuration.md)
+    + pd-server
+      - [Configuration Flags](op-guide/pd-configuration.md)
+    + tikv-server
+      - [Configuration Flags](op-guide/tikv-configuration.md)
+  + Tools
+    - [Syncer](tools/syncer.md)
+    - [mydumper](tools/mydumper.md)
+    - [Loader](tools/loader.md)
+    + Data Migration
+      - [Overview](tools/dm/overview.md)
+      - [Restrictions](tools/dm/overview.md#usage-restrictions)
+      - [Deploy](tools/dm/practice.md)
+      + Features
+        - [Table Routing](tools/dm/data-synchronization-features.md#table-routing)
+        - [Black and White Lists](tools/dm/data-synchronization-features.md#black-and-white-table-lists)
+        - [Binlog Event Filtering](tools/dm/data-synchronization-features.md#binlog-event-filtering)
+        - [Column Mapping](tools/dm/data-synchronization-features.md#column-mapping)
+        - [Synchronization Delay Monitoring](tools/dm/data-synchronization-features.md#synchronization-delay-monitoring)
+        + Sharding Support
+          - [Introduction](tools/dm/shard-merge.md)
+          - [Restrictions](tools/dm/shard-merge.md#restrictions)
+          - [Handle Sharding DDL Locks Manually](tools/dm/manually-handling-sharding-ddl-locks.md)
+      + Usage Scenarios
+        - [Simple Scenario](tools/dm/simple-synchronization-scenario.md)
+        - [Shard Merge Scenario](tools/dm/shard-merge-scenario.md)
+      + Configure
+        - [Overview](tools/dm/dm-configuration-file-overview.md)
+        - [Task Configuration](tools/dm/task-configuration-file-intro.md)
+      - [Monitor](tools/dm/monitor.md)
+      - [Manage the Task](tools/dm/manage-task.md)
+      - [Cluster Operations](tools/dm/cluster-operations.md)
+    + TiDB-Lightning
+      - [Overview](tools/lightning/overview-architecture.md)
+      - [Deployment](tools/lightning/deployment.md)
+      - [Checkpoints](tools/lightning/checkpoints.md)
+      - [Table Filter](tools/lightning/filter.md)
+      - [CSV Support](tools/lightning/csv.md)
+      - [Monitor](tools/lightning/monitor.md)
+    - [TiDB-Binlog](tools/tidb-binlog-cluster.md) (Enterprise)
+    - [PD Control](tools/pd-control.md)
+    - [PD Recover](tools/pd-recover.md)
+    - [TiKV Control](https://github.com/tikv/tikv/blob/master/docs/tools/tikv-control.md)
+    - [TiDB Controller](tools/tidb-controller.md)
+    - [Download](tools/download.md)
+  - [Roadmap](ROADMAP.md)
+  - [Adopters](adopters.md)
++ FAQs
+  - [TiDB Server FAQs](FAQ.md)
```

Comment:
It's not only about TiDB Server. If we have the FAQ for TiDB Server, users would also expect one of PD Server, TiKV Server.

```suggestion
  - [TiDB FAQs](FAQ.md)
```